src/query/generated/assets/openapi/spec.yml

```diff
@@ -339,6 +339,35 @@ paths:
           description: ""
           schema:
             $ref: "#/definitions/GenericError"
+  /services/m3db/placement/replace:
+    post:
+      tags:
+      - "M3DB Placement"
+      - "M3DB"
+      summary: "Replace an M3DBNode"
```

Comment:
do we concatenate M3DB and Node together in the other summaries?

Reply:
Nope, will fix

Reply:
Other M3DB ones:
```
 summary: "Add an instance to the placement"
```

Maybe "Replace an instance in the placement"? *shrug*